src/expenses.router.js

```diff
@@ -0,0 +1,85 @@
+const { Router } = require('express');
+const service = require('./expensesServices.js');
+const expensesRouter = Router();
+
+expensesRouter.get('/', async (req, res) => {
```

Comment:
```suggestion
expensesRouter.get('/', getAllExpenses)
```
Consider creating a controller for processing request